prismic-model/js/exhibitions.js

```diff
@@ -32,8 +32,34 @@ const Exhibitions = {
   Promo: {
     promo
   },
-  // When we get a better relational model up, this will be deprecated
-  'Related promos': {
+  Resources: {
+    resources: list('Resources', {
+      resource: link('Resource', 'document', ['exhibition-resources'])
+    })
+  },
+  Related: {
+    relatedOfs: list('Related of', {
+      relatedOf: link('In this exhibition', 'document', ['installations', 'events'])
+    }),
+    relatedAbouts: list('Related about', {
```

Comment:
As per discussions (and this is just me reiterating, so if we think it's good, we're good) - but talking to @silveroliver, we're not saying it's related. We know the explicit relationship, so should expose it as such.

Installations for example don't go in here, exhibitions are exhibits within an exhibition, which could also have works exhibited in it for example.